tools/flang1/utils/symtab/sharedefs.h

```diff
@@ -30,21 +30,21 @@
 
 /* declare:
  *  struct{
- *     dt* stg_base;
+ *     dt* stg_base, *stg_align;
```

Comment:
`STG_MEMBERS` is instantiated in multiple data structures (twice in `STB` and three times in `ASTB`, among others), but it seems that only `stb.dt.stg_align` is ever used. Did you consider another way to declare this array that could have less impact on unrelated types?

Reply:
> `STG_MEMBERS` is instantiated in multiple data structures (twice in `STB` and three times in `ASTB`, among others), but it seems that only `stb.dt.stg_align` is ever used. Did you consider another way to declare this array that could have less impact on unrelated types?

Yes, you are right. I declare this array in `STB` structure as `  struct{
    STG_MEMBERS(ISZ_T);
  }dt_align;` field in the latest patches.